esp/esp/program/modules/handlers/studentlunchselection.py

```diff
@@ -120,6 +120,7 @@ def isCompleted(self):
     @main_call
     @needs_student
     @meets_cap
+    @meets_deadline('/Classes')
```

Comment:
One option would be to use, say, `/Classes/Lunch`, so that by default both would open at once but it would be possible to just open lunch.